src/main/target/common_post.h

```diff
@@ -192,6 +202,11 @@
 #undef USE_USB_MSC
 #endif
 
+#if !defined(USE_SDCARD)
```

Comment:
For maintainability's sake I'd move this to right above where the rest of the SDCARD related defines are.